app/src/pages/inside/testItemPage/testItemPage.jsx

```diff
@@ -106,31 +104,25 @@ const messages = defineMessages({
 export const getDeleteItemsActionParameters = (
   items,
   formatMessage,
-  { userId, parentLaunch, ...rest } = {},
-) => {
-  const isOwner = isItemOwner(userId, items[0], parentLaunch);
-
-  return {
-    header:
-      items.length === 1
-        ? formatMessage(messages.deleteModalHeader)
-        : formatMessage(messages.deleteModalMultipleHeader),
-    mainContent:
-      items.length === 1
-        ? formatMessage(messages.deleteModalContent, {
-            b: (data) => DOMPurify.sanitize(`<b>${data}</b>`),
-            name: items[0].name,
-          })
-        : formatMessage(messages.deleteModalMultipleContent),
-    warning:
-      (!isOwner &&
-        (items.length === 1
-          ? formatMessage(messages.warning)
-          : formatMessage(messages.warningMultiple))) ||
-      '',
-    ...rest,
-  };
-};
+  { parentLaunch, ...rest } = {},
+) => ({
+  header:
+    items.length === 1
+      ? formatMessage(messages.deleteModalHeader)
+      : formatMessage(messages.deleteModalMultipleHeader),
+  mainContent:
+    items.length === 1
+      ? formatMessage(messages.deleteModalContent, {
+          b: (data) => DOMPurify.sanitize(`<b>${data}</b>`),
+          name: items[0].name,
+        })
+      : formatMessage(messages.deleteModalMultipleContent),
+  warning:
+    (items.length === 1
+      ? formatMessage(messages.warning)
+      : formatMessage(messages.warningMultiple)) || '',
```

Comment:
Do we need to remove the 'own'/'not own' mentions from the `warning` and `warningMultiple` messages?
Please discuss with BA.

Reply:
Deleted